andela/hc-logros-infinitos
# Ft alert often job 153728011

## What does this PR do?*

• Add the alert functionality when the job runs often and the test to ensure it works.

## Description of task to be completed

• The functionality enables the user to get alerts when the job is running too often.

## How should this be manually tested

• Add a check, ping it the first time to be up, then ping it second time before the reversed grace time , then it will alert you that it is running often.

## Any background context you want to provide

• Need bootstrap for some badges and lables

## What are the relevant pivotal Tracker stories*

• [#153728011](https://www.pivotaltracker.com/n/projects/2137217)
